ui/static/ui/js/require_config.js

```diff
@@ -11,7 +11,8 @@ var REQUIRE_PATHS = {
   csrf: "../ui/js/csrf",
   facets: "../ui/js/facets",
   listing: "../ui/js/listing",
-  setup_manage_taxonomies: "../ui/js/manage_taxonomies.jsx?noext"
+  setup_manage_taxonomies: "../ui/js/manage_taxonomies.jsx?noext",
+  jquery_mockjax: "../node_modules/jquery-mockjax/src/jquery.mockjax"
```

Comment:
This line should be removed, the path is already added to `test-main.js` which sets up paths specific to the tests. We shouldn't ever need the jQuery mocking library outside of our tests.